core/src/main/scala/akka/persistence/cassandra/query/EventsByTagStage.scala

```diff
@@ -433,10 +432,21 @@ import scala.compat.java8.FutureConverters._
       private def scanForDelayedEvents(): Unit = {
         if (!stageState.delayedScanInProgress) {
           updateStageState(_.copy(delayedScanInProgress = true))
-          val startOfScan = settings.eventsByTagBacktrack.period match {
-            case Max             => Uuids.startOf(stageState.currentTimeBucket.previous(1).key)
-            case Fixed(duration) => Uuids.startOf((Uuids.unixTimestamp(stageState.fromOffset) - duration.toMillis))
-          }
+
+          val startOfPreviousBucket = stageState.currentTimeBucket.previous(1).key
+          val currentTime = Uuids.unixTimestamp(stageState.fromOffset)
+          val (startOfScan) =
+            if (System.currentTimeMillis() - stageState.previousLongDelayedScan > backtracking.longIntervalMillis()) {
```

Comment:
can we simplify by not having the `long-interval` config? this can be checked for each `interval`

Reply:
Sorry I don't follow, then how often would be do the expensive scan? Do you mean each long-period?

Reply:
Yep it is read into the BackTrackConfig https://github.com/chbatey/akka-persistence-cassandra/blob/issue-606/core/src/main/scala/akka/persistence/cassandra/query/CassandraReadJournalConfig.scala#L183

and then used to decide how frequently the long backtrack is done

Reply:
Got it, my misunderstanding